packages/libp2p/src/identify/index.ts

```diff
@@ -60,6 +66,22 @@ export const multicodecs = {
 
 export const Message = { Identify }
 
+export interface IdentifyService {
+  /**
+   * due to the default limits on inbound/outbound streams for this protocol,
+   * invoking this method when runOnConnectionOpen is true can lead to unpredictable results
+   * as streams may be closed by the local or the remote node.
+   * Please use with caution. If you find yourself needing to call this method to discover other peers that support your protocol,
+   * you may be better off configuring a topology to be notified instead.
+   *
+   * @param connection
+   * @param options
+   * @returns
```

Comment:
These jsdoc comments don't seem to add any information that isn't derivable from the interface?

```suggestion
```